src/python/base/tasks.py

```diff
@@ -376,12 +376,16 @@ def add_task(command, argument, job_type, queue=None, wait_time=None):
       [task.to_pubsub_message()])
 
 
+def get_task_lease_timeout():
+  """Return the task lease timeout."""
+  return environment.get_value('TASK_LEASE_SECONDS', TASK_LEASE_SECONDS)
```

Comment:
should this be restricted to fuzzing tasks only? seems like there is an inconsistency here since because whatever is in TASK_LEASE_SECONDS_BY_COMMAND (i.e. regression, corpus_pruning) may be unintentionally overridden by this.

Reply:
Do you have a fix in mind ? I ideally wanted this configurable in job definition [https://github.com/google/clusterfuzz/issues/2028] but this code gets triggered much before.

Reply:
actually never mind, i misread the code before. TASK_LEASE_SECONDS_BY_COMMAND still has precedence, so I don't have any more concerns here :)